asteroid/dsp/overlap_add.py

```diff
@@ -112,11 +112,14 @@ def ola_forward(self, x):
                 # we determine best perm based on xcorr with previous sources
                 frame = _reorder_sources(frame, out[-1], n_src, self.window_size, self.hop_size)
 
+            out.append(frame)
+
+        # apply windowing/scaling *after* _reorder_sources has been called.
+        for frame in out:
             if self.use_window:
-                frame = frame * self.window.to(frame)
+                frame *= self.window.to(frame)
```

Comment:
This is not a functional change, only cosmetic.

Reply:
If I am not mistaken, `*=` acts in-place, so the `frame` tensor itself gets modified, meaning that the `frame` in the `out` list is also modified.

`frame = frame * self.window.to(frame)` would not accomplish this, would it?

Reply:
Indeed, I misread the logic at first, thanks.